scripts/tracking/SiamRPN_demo.py

```diff
@@ -0,0 +1,82 @@
+"""01. Predict with pre-trained SiamRPN models
```

Comment:
This whole file is already under `tutorials/tracking`, so you can delete this file at this location.